doc/dialects/teradata.md

```diff
@@ -1,43 +1,80 @@
 
-Please also upload the jar files to a bucket for the adapter script.
+Now register the driver in EXAOperation:
 
-## Adapter script
+1. Click "Software"
+1. Switch to tab "JDBC Drivers"
+1. Click "Browse..."
+1. Select JDBC driver file
+1. Click "Upload"
+1. Click "Add"
+1. In dialog "Add EXACluster JDBC driver" configure the JDBC driver (see below)
+
+You need to specify the following settings when adding the JDBC driver via EXAOperation.
+
+| Parameter | Value                                               |
+|-----------|-----------------------------------------------------|
+| Name      | `TERADATA`                                          |
+| Main      | `com.teradata.jdbc.TeraDriver`                      |
+| Prefix    | `jdbc:teradata:`                                    |
+| Files     | `terajdbc4.jar`, `tdgssconfig.jar`                  |
+
+## Uploading the JDBC Driver to EXAOperation
+
+1. [Create a bucket in BucketFS](https://docs.exasol.com/administration/on-premise/bucketfs/create_new_bucket_in_bucketfs_service.htm)
+1. Upload the driver to BucketFS
+
+This step is necessary since the UDF container the adapter runs in has no access to the JDBC drivers installed via EXAOperation but it can access BucketFS.
+
+## Installing the Adapter Script
+
+Upload the last available release of [Virtual Schema JDBC Adapter](https://github.com/exasol/virtual-schemas/releases) to Bucket FS.
+
+Then create a schema to hold the adapter script.
 
 ```sql
-CREATE OR REPLACE JAVA ADAPTER SCRIPT adapter.jdbc_adapter 
-  AS
-  
-  // This is the class implementing the callback method of the adapter script
-  %scriptclass com.exasol.adapter.jdbc.JdbcAdapter;
-
-  // This will add the adapter jar to the classpath so that it can be used inside the adapter script
-  // Replace the names of the bucketfs and the bucket with the ones you used.
-  %jar /buckets/bucketfs1/bucket1/virtualschema-jdbc-adapter-dist-1.19.7.jar;
-									 
-  // You have to add all files of the data source jdbc driver here (e.g. MySQL or Hive)
-  %jar /buckets/bucketfs1/bucket1/terajdbc4.jar;
-  %jar /buckets/bucketfs1/bucket1/tdgssconfig.jar;
+CREATE SCHEMA ADAPTER;
+```
+
+The SQL statement below creates the adapter script, defines the Java class that serves as entry point and tells the UDF framework where to find the libraries (JAR files) for Virtual Schema and database driver.
 
+
+```sql
+CREATE OR REPLACE JAVA ADAPTER SCRIPT ADAPTER.JDBC_ADAPTER AS
+  %scriptclass com.exasol.adapter.RequestDispatcher;
+  %jar /buckets/<BFS service>/<bucket>/virtualschema-jdbc-adapter-dist-1.19.1.jar;
+  %jar /buckets/<BFS service>/<bucket>/terajdbc4.jar;
+  %jar /buckets/<BFS service>/<bucket>/tdgssconfig.jar;
 /
+;
+```
+
+## Defining a Named Connection
+
+Define the connection to Teradata as shown below. 
+
+```sql
+CREATE OR REPLACE CONNECTION TERADATA_CONNECTION
+TO 'jdbc:teradata://<Database server>/database=<Database name>'
```

Comment:
```suggestion
TO 'jdbc:teradata://<database host>/database=<database name>'
```

Reply:
No port?

Reply:
Fixed. I haven't seen port on the example...